pkg/nack/receive_log_test.go

```diff
@@ -8,6 +8,8 @@ import (
 
 func TestReceivedBuffer(t *testing.T) {
 	for _, start := range []uint16{0, 1, 127, 128, 129, 511, 512, 513, 32767, 32768, 32769, 65407, 65408, 65409, 65534, 65535} {
+		start := start
```

Comment:
Looks good @mengelbart . I had removed this line earlier in the day in another PR as it looked redundant. Is this needed?

Reply:
I think it depends on the test case: `start` will be overwritten in each iteration, so if the function called in the loop uses `start` after the iteration is finished, it will operate on the value of the next iteration. Using `start := start` copies the value to a new variable. Not sure if this is a real issue here, but better be safe and `golangci-lint` will complain about it.

Reply:
Got it. I removed it and the tests passed. So thought it was okay to remove.